sdk/notificationhubs/notification-hubs/review/notification-hubs.api.md

```diff
@@ -508,6 +558,143 @@ export interface FirebaseLegacyWebNativePayload {
     title?: string;
 }
 
+// @public
+export interface FirebaseV1AndroidConfig {
+    collapse_key?: string;
+    data?: Record<string, any>;
+    direct_boot_ok?: boolean;
+    fcm_options?: FirebaseV1AndroidFcmOptions;
+    notification?: FirebaseV1AndroidNotification;
+    priority: "normal" | "high";
+    restricted_package_name?: string;
+    ttl?: string;
+}
+
+// @public
+export interface FirebaseV1AndroidFcmOptions {
+    analytics_label?: string;
+}
+
+// @public
+export interface FirebaseV1AndroidNotification {
+    body?: string;
+    body_loc_args?: string[];
+    body_loc_key?: string;
+    channel_id?: string;
+    click_action?: string;
+    color?: string;
+    default_light_settings?: boolean;
+    default_sound?: boolean;
+    default_vibrate_timings?: boolean;
+    event_time?: string;
+    icon?: string;
+    image?: string;
+    light_settings?: {
+        color: {
+            red: number;
+            green: number;
+            blue: number;
+            alpha: number;
+        };
+        light_on_duration: string;
+        light_off_duration: string;
+    };
+    local_only?: boolean;
+    notification_count?: number;
+    // (undocumented)
+    notification_priority?: "PRIORITY_UNSPECIFIED" | "PRIORITY_MIN" | "PRIORITY_LOW" | "PRIORITY_DEFAULT" | "PRIORITY_HIGH" | "PRIORITY_MAX";
+    sound?: string;
+    sticky?: boolean;
+    tag?: string;
+    ticker?: string;
+    title?: string;
+    title_loc_args?: string[];
+    title_loc_key?: string;
+    vibrate_timings?: string[];
+    visibility?: "VISIBILITY_UNSPECIFIED" | "PRIVATE" | "PUBLIC" | "SECRET";
+}
+
+// @public
+export interface FirebaseV1ApnsConfig {
+    fcm_options?: FirebaseV1ApnsFcmOptions;
+    headers?: Record<string, string>;
+    payload?: AppleNativeMessage;
+}
+
+// @public
+export interface FirebaseV1ApnsFcmOptions {
```

Comment:
Curious about your opinion - do you think we want to add Apns and WebPush options? ANH doesn't support APNS and Webpush via ANH FcmV1 operations.